WcaOnRails/app/assets/stylesheets/edit_schedule.scss

```diff
@@ -0,0 +1,223 @@
+#edit-schedule-area {
+  .panel-primary {
+    .collapse-indicator {
+      color: #fff;
+    }
+  }
+
+  #venues-edit-panel {
+    .venue-map {
+      height: 300px;
+    }
+    .venue-form-label {
+      @extend label;
+    }
+    .panel-venue {
+      @extend .col-xs-12, .col-md-6, .col-lg-4;
+      margin-bottom: 15px;
+      .new-venue-link {
+        width: 100%;
+      }
+      .venue-title {
+        padding-top: 7px;
+      }
+      .panel-body {
+        .row {
+          margin-bottom: 10px;
+        }
+      }
+    }
+  }
+
+  #schedules-edit-panel {
+    #activity-picker-panel {
+      &.affix {
+        top: 10px;
+      }
+      &.affix-bottom {
+        position: absolute;
+        bottom: auto;
+      }
+      .selected-activity {
+        border: 2px solid #791f82;
```

Comment:
Could you pull all colors out of this file, give them names, and put them in variables.scss instead?